app/controllers/api/v1/offers_controller.rb

```diff
@@ -98,6 +98,15 @@ def complete_review
       render json: @offer, serializer: serializer
     end
 
+    api :PUT, '/v1/offers/1/close_offer', "Mark Offer as closed."
+    param :offer, Hash, required: true do
+      param :state_event, String, "State transition event ex: 'close'", required: true
+    end
+    def close_offer
+      @offer.update_attributes(review_offer_params)
+      render json: @offer, serializer: serializer
+    end
+
```

Comment:
@swatijadhav ```complete_review``` and ```close_offer``` are exactly the same method. Could you DRY the up a bit? Thanks